tests/InsertionsClient.Console.Test/InputParsingTests.cs

```diff
@@ -89,5 +89,101 @@ public void TestLoadedIgnoredPackages()
             Assert.IsTrue(ignoredPackages.SetEquals(results.IgnoredNuGets), $"Mismatched ignore packages");
             Assert.IsFalse(results.UpdatedNuGets.Any(n => ignoredPackages.Contains(n.PackageId)), "Packages that shouldn't have been updated were updated.");
         }
+
+        /// <summary>
+        /// Tests whether <see cref="InputLoading.LoadManifestPaths(string, out int)"/> can load a single valid manifest file.
+        /// </summary>
+        [TestMethod]
+        public void TestManifestPathLoading()
```

Comment:
does it make sense to make a test for at least 2 manifests successful load? Now only one manifest processing is tested.

Reply:
Added one more test case.